src/input/cursor.c

```diff
@@ -140,6 +140,14 @@ request_cursor_notify(struct wl_listener *listener, void *data)
 		return;
 	}
 
+	if (!wl_list_empty(&seat->server->tablet_manager->clients)) {
```

Comment:
note to self: TODO: this prevents the drag cursor going back to a normal cursor when finishing moving a CSD application.

Reply:
Remove that part. While it fixed the hovering into a GTK4 app that wants a different cursor when entering its surface (thus when somewhat still being over de resize area), it also prevented setting a cursor in XWayland apps without tablet support (e.g. VS Code) next to the remaining cursor after moving.

Reply:
Re-added, somehow still makes sense to prevent those conflicts.Mitigated by resetting the cursor in interactive_cancel. Let's play some more.